packages/expo-video/src/VideoPlayerEvents.types.ts

```diff
@@ -0,0 +1,176 @@
+ */
+export type StatusChangeEventPayload = {
+  /**
+   * New status of the player.
+   */
+  status: VideoPlayerStatus;
+
+  /**
+   * Previous status of the player.
+   */
+  oldStatus?: VideoPlayerStatus;
+
+  /**
+   * Error object containing information about the error that occurred.
+   */
+  error?: PlayerError;
+};
+
+/**
+ * Data delivered with the [`playingChange`](#videoplayerevents) event.
+ */
+export type PlayingChangeEventPayload = {
+  /**
+   * Boolean value whether the player is currently playing.
+   */
+  isPlaying: boolean;
+
+  /**
+   * Previous value of the `isPlaying` property.
+   */
+  oldIsPlaying?: boolean;
+};
+
+/**
+ * Data delivered with the [`playbackRateChange`](#videoplayerevents) event.
+ */
+export type PlaybackRateChangeEventPayload = {
+  /**
+   * Float value indicating the current playback speed of the player.
+   */
+  playbackRate: number;
+
+  /**
+   * Previous value of the `playbackRate` property.
+   */
+  oldPlaybackRate?: number;
+};
+
+/**
+ * Data delivered with the [`volumeChange`](#videoplayerevents) event.
+ */
+export type VolumeChangeEventPayload = {
+  /**
+   * Float value indicating the current volume of the player.
+   */
+  volume: number;
+
+  /**
+   * Previous value of the `volume` property.
+   */
+  oldVolume?: number;
+};
+
+/**
+ * Data delivered with the [`mutedChange`](#videoplayerevents) event.
+ */
+export type MutedChangeEventPayload = {
+  /**
+   * Boolean value whether the player is currently muted.
+   */
+  muted: boolean;
+
+  /**
+   * Previous value of the `isMuted` property.
+   */
+  oldMuted?: boolean;
+};
+
+/**
+ * Data delivered with the [`sourceChange`](#videoplayerevents) event.
+ */
+export type SourceChangeEventPayload = {
+  /**
+   * New source of the player.
+   */
+  source: VideoSource;
+
+  /**
+   * Previous source of the player.
+   */
+  oldSource?: VideoSource;
+};
+
+/**
+ * Data delivered with the [`timeUpdate`](#videoplayerevents) event, contains information about the current playback progress.
+ */
+export type TimeUpdateEventPayload = {
+  /**
+   * Float value indicating the current playback time in seconds. Same as the [`currentTime`](#currenttime) property.
+   */
+  currentTime: number;
+
+  /**
+   * The exact timestamp when the currently displayed video frame was sent from the server,
+   * based on the `EXT-X-PROGRAM-DATE-TIME` tag in the livestream metadata.
+   * Same as the [`currentLiveTimestamp`](#currentlivetimestamp) property.
+   * @platform android
+   * @platform ios
+   */
+  currentLiveTimestamp: number | null;
+
+  /**
+   * Float value indicating the latency of the live stream in seconds.
+   * Same as the [`currentOffsetFromLive`](#currentoffsetfromlive) property.
+   * @platform android
+   * @platform ios
+   */
+  currentOffsetFromLive: number | null;
+
+  /**
+   * Float value indicating how far the player has buffered the video in seconds
+   * Same as the [`bufferedPosition`](#bufferetPosition) property
```

Comment:
```suggestion
   * Float value indicating how far the player has buffered the video in seconds.
   * Same as the [`bufferedPosition`](#bufferedposition) property.
```

Let's fix the typo in the internal link and add the missing period in both sentences.

Reply:
@behenate This typo has been fixed in https://github.com/expo/expo/pull/32052, which means that the PR need a rebase, since it regenerated the API data based on outdated code.